src/sandstorm/ip.capnp

```diff
@@ -50,6 +50,21 @@ interface IpNetwork @0xa982576b7a2a2040 {
   # access". The IpNetwork capability is a dangerous capability that should only be granted to
   # trusted drivers. Only the Sandstorm server administrator is likely to possess this capability.
 
+  struct PowerboxTag {
+    # Tag to be used in a `PowerboxDescriptor` to describe an `IpNetwork`.
+
+    encryption :union {
```

Comment:
Good point!

I've now boxed the union into an `Encyrption` struct, but I've kept it as a field of `IpNetwork.PowerboxTag` rather than adding it as an independent tag, mostly because doing so fits better with our current `FrontendRefRegistry` code. I've added a note explaining that other interfaces may use `Encryption` as an independent tag. Hopefully, most such capabilities can be implemented in userland and hence will not need to have a `frontendRef` representation.